labs/SAW/SAW.md

```diff
@@ -1665,11 +1741,11 @@ sprite_p = self.alloc( alias_ty("struct.sprite_t")
 Notice that we use 3 double quotes `"""` in our `cry_f` call. This technique is
 useful when we want to separate our expected Cryptol-defined behaviors over 
 multiple lines to improve code readability. Python considers the 3 double 
-quotes as a multiline string. Multiline strings may also be used as block 
+quotes as a multi-line string. Multi-line strings may also be used as block 
 comments in Python.
 
 However, the setup we see above results in wide open spaces, which will be 
-noticable when debugging strings passed to the family of `cry` functions. These 
+noticeable when debugging strings passed to the family of `cry` functions. These 
 spaces can be mitigated using `dedent` from the `textwrap` package that comes 
 with Python. For example:
 

```

Comment:
Should this be `cry_f` too?

Reply:
The `sprite_t`  example needs `cry_f` since it takes in Python local variables while the `dedent` example doesn't, so it can just use `cry`.